docs/docs/integrations/tools/azure_content_safety.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
this is a nice guide! few comments:

- could you add a URL to the azure context safety API / docs? (i see you have it in the docstring)
- could you add `getpass()` for environment variables to allow users to set them manually, like here https://python.langchain.com/docs/integrations/tools/bing_search/
- for the agent example, could you please use 

```python
from langgraph.prebuilt import create_react_agent

agent = create_react_agent(model, tools)
agent.invoke({"messages": [("user",  f"Can you check the following text for harmful content : {input}")]})
```
this will also allow you to get rid of the structured chat prompt
- could you remove the chaining section?